WordPress/src/main/java/org/wordpress/android/ui/stats/StatsActivity.java

```diff
@@ -621,8 +622,8 @@ private void bumpPromoAnalyticsAndShowPromoDialogIfNecessary() {
         AppPrefs.bumpAnalyticsForStatsWidgetPromo();
 
         // Should we display the widget promo?
-        int counter = AppPrefs.getAnalyticsForStatsWidgetPromo();
-        if (counter == 3 || counter == 1000 || counter == 10000) {
+
+        if (shouldShouldWidgetPromo()) {
```

Comment:
I think you meant `shouldShowWidgetPromo()`?

Reply:
... yes, thanks for that.